cdap-docs/integrations/source/index.rst

```diff
@@ -38,7 +44,7 @@ Integrations
 .. rubric:: Ambari
 
 .. |ambari| replace:: **Configuring and Installing:**
-.. _ambari: ../admin-manual/installation/ambari.html
+.. _ambari: /../admin-manual/installation/ambari.html
```

Comment:
Yes. There are two ways of doing links in reST: one is absolute (starting with a "/") and one is relative (not starting with a "/'). Absolute links are rooted in the start of the source for a particular set of documents (in our case, the ``source`` directory), so the advantage of such as a link is that ``/../admin-manual`` will work *anywhere* in the *integrations* manual, and point to the same place, without needing to change it if you go up or down in the hierarchy.

There is a third method of linking, unique to Sphinx, using the ``:ref:`` syntax...but that's another story, which you are probably familiar with.